index.bs

```diff
@@ -190,6 +190,32 @@ extensible.
 </script>
 ```
 
+## Display Arbitrary Content ## {#example-arbitrary-content}
+
+```html
+<div id="interactive-ui"></div>
+<div id="non-interactive-ui"></div>
```

Comment:
This ends up being rendered on one line. Should you add double spaces to have a line break?

Reply:
Done.

Reply:
Given that the "Preview" and "Diff" have a different rendering, I'm not sure if I was actually correct now :(